src/rules/keywords.md

```diff
@@ -0,0 +1,14 @@
+# Keywords
+
+## Concentration
+
+Some spells and other effects require you to maintain concentration in order to keep them active. If you lose concentration, such an effect ends.
+
+If a spell must be maintained with concentration, that fact appears in its Duration entry, and the spell specifies how long you can concentrate on it. You can end concentration on any such effect at any time (no action required).
+
+Normal activity, such as moving and attacking, doesn't interfere with concentration. The following factors can break concentration:
+
+- Invoking another effect that requires concentration. You lose concentration if you invoke another effect that requires concentration. You can't concentrate on two effects at once.
+- Taking damage. Whenever you take damage while you are concentrating, you must make a Constitution saving throw to maintain your concentration. The DC equals 10 or half the damage you take, whichever number is higher. If you take damage from multiple sources, such as an arrow and a dragon's breath, you make a separate saving throw for each source of damage.
+- Being incapacitated or killed. You lose concentration on an effect if you are incapacitated or if you die.
```

Comment:
Are the first sentences here supposed to be in bold? And the last bullet its own paragraph?

Reply:
They are indeed.